packages/auth/src/exports_auth.js

```diff
@@ -687,34 +670,43 @@ fireauth.exportlib.exportFunction(
     fireauth.exportlib.exportFunction(namespace,
         'ActionCodeURL', fireauth.ActionCodeURL, []);
 
-    // Register Auth service with firebase.App.
-    firebase.INTERNAL.registerService(
-        fireauth.exportlib.AUTH_TYPE,
-        factory,
-        namespace,
-        // Initialize Auth when an App is created, so that tokens and Auth state
-        // listeners are available.
-        function (event, app) {
-          if (event === 'create') {
-            try {
-              app[fireauth.exportlib.AUTH_TYPE]();
-            } catch (e) {
-              // This is a silent operation in the background. If the auth
-              // initialization fails, it should not cause a fatal error.
-              // Instead when the developer tries to initialize again manually,
-              // the error will be thrown.
-              // One specific use case here is the initialization for the nodejs
-              // client when no API key is provided. This is commonly used
-              // for unauthenticated database access.
-            }
-          }
-        }
-        );
-
+    // Create auth components to register with firebase
+    const authComponent = { // provides Auth public APIs
```

Comment:
Preferably move comment to newline. Capitalize first letter and add dot to end of line.
`// Provides Auth public APIs.`

Reply:
Done.